packages/application-shell/src/components/navbar/navbar-old/navbar.styles.js

```diff
@@ -0,0 +1,11 @@
+const fs = require('fs');
```

Comment:
I think this would be a good time to learn why we have this setup here.

I don't see any other setup like this one for styles in the repository so it would be great to understand why we need it here.

@emmenko do you have more context about this? 🙏

Reply:
This is due to the fact that the navbar is the only component that uses CSS modules (between App Kit and UI Kit) instead of Emotion, and this poses a problem with bundling.

Therefore, we use a Babel macro to inject the styles into the JS bundle so that we don't need to manage and load external CSS files.

That's why I wanted us to refactor the navbar implementation, so that we can use Emotion and remove this hack.

Reply:
Thanks for the explanation 👍